endpoint_storage/src/openapi_spec.yml

```diff
@@ -0,0 +1,140 @@
+openapi: "3.0.2"
+info:
+  title: Endpoint Storage API
+  description: Endpoint Storage API
+  version: "1.0"
+  license:
+    name: "Apache"
+    url: https://github.com/neondatabase/neon/blob/main/LICENSE
+servers:
+  - url: ""
+paths:
+  /status:
+    description: Healthcheck endpoint
+    get:
+      description: Healthcheck
+      security: []
+      responses:
+        "200":
+          description: OK
+
+  /{tenant_id}/{timeline_id}/{endpoint_id}/{key}:
```

Comment:
Just noticed, as an FYI without static identifiers between each layer. It's going to be really hard to extend the API without introducing inconsistencies in naming schemes.